content/trial_getting_started/examples/basic/aws_basics.md

```diff
@@ -176,74 +164,104 @@ The blueprint can be examined in the Cloudify UI, however in this case
+1. Upload the blueprint describing the service to the Cloudify Manager.
+1. Create a deployment from that uploaded blueprint - this generates a model of the service topology in the Cloudify Database
+1. Run the install workflow for that created deployment to apply the model to the infrastructure.
 
-`6`. Paste the URL of the blueprint package in the URL field. Provide any name you like.
+Let's run these one by one.
 
-`7`. Select aws.yaml from the Blueprint YAML file menu.
-     (You can leave the Blueprint icon field blank. It is only for decoration).
+To upload a blueprint to the Cloudify manager using the Management Console UI, select the **Local Blueprints** page, and use the **Upload** button.
 
-`8`. Click **Upload**.
+* Blueprint package: https://github.com/cloudify-community/blueprint-examples/releases/download/5.0.5-9/virtual-machine.zip
+* Blueprint name: virtual-machine
+* Blueprint YAML file: aws.yaml
 
-The blueprint should appear in the blueprint list under the name you provided.
+Once the blueprint is uploaded, it will be displayed in the Blueprints widget. to deploy the blueprint click the **Create deployment** button next to the blueprint you wish to deploy. Specify a deployment name, and click **Deploy**
 
-`9`. On the right, you will see a rocket icon. click the rocket icon and create deployment dialog will be shown.
+Switch to the **Deployments** page. The deployment you have created should be displayed in the deployments list.
 
-`10`. Provide a name you like in the Deployment name field.
+To apply the deployment and push it to the infrastructure run the **Install** workflow by clicking the **Execute workflow** menu next to the deployment and selecting **Install**.
 
-`11`. You can skip the Site name field.
+You can track the progress of the installation workflow by checking the node instances progress, or get a detailed view by clicking the deployment, and in the drill down page scroll down to the **Deployment Executions** widget and expand the **Install** workflow.
 
-`12`. Provide values for any inputs that you would like to change.
+### Step 4: Check your orchestrated services
 
-`13`. Click **Deploy**.
+In this example we have setup a simple infrastructure. An EC2 instance was created in the region specified in the secrets, alongside VPC and various other nodes.
 
-The newly created deployment should appear in the deployment list under the name you provided.
+* Go to your AWS console and see the EC2 instance and other instances that were created.
+* Examine the deployment page in the Management Console for more information about your deployed nodes, the topology, and the installation logs.
 
-`14`. Go to Deployments and press on your deployment, then press **Execute workflow->Default workflows->Install**
+## Step 5: OK, I am done, how do I tear it down?
 
-You did it!
+To remove the deployment and delete all resources from AWS simply run the **Uninstall workflow**
```

Comment:
need also to delete the deployment and the blueprint after running the uninstall workflow .

Reply:
fixed.